src/stories/containers/Finances/components/SectionPages/BreakdownTable/useBreakdownTable.tsx

```diff
@@ -101,7 +101,8 @@ export const useBreakdownTable = () => {
     : ['Annually', 'Quarterly', 'Monthly'];
 
   const handleChangeYears = (value: string) => {
-    setYears(value);
+    setYear(value);
+    router.push(`/finances?year=${value}` /* undefined, { shallow: true } */);
```

Comment:
I recommend no to use hardcoded routes as they may change, instead, build the route using the `siteRoutes` helper:

```ts
// this may not work exactly as it is :)
router.push({
    pathname: siteRoutes.finances,
    query: {
        ...router.query,
        year: value
    }
},
undefined,
{ shallow: true }
)
```

Reply:
Done! check: https://github.com/makerdao-ses/ecosystem-dashboard/pull/1117/commits/dcb1808c34b8389cc547be03221a06f17d27209d